pydis_site/apps/content/resources/guides/python-guides/discord-messages-with-colors.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Discord messages with colors
```

Comment:
A little nitpick to keep in consistency with the other titles
```suggestion
title: Discord Messages with Colors
```

Reply:
This would also apply to the PR #690 right?